ports/libebur128/portfile.cmake

```
@@ -0,0 +1,20 @@
+if((VCPKG_TARGET_ARCHITECTURE STREQUAL "arm" OR VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64") AND VCPKG_CMAKE_SYSTEM_NAME STREQUAL "")
```

```suggestion
if((VCPKG_TARGET_ARCHITECTURE STREQUAL "arm" OR VCPKG_TARGET_ARCHITECTURE STREQUAL "arm64") AND VCPKG_TARGET_IS_WINDOWS)
```
Use `VCPKG_TARGET_IS_WINDOWS` to express Windows platform.